controller/src/command.cpp

```diff
@@ -264,18 +267,13 @@ void command_responseSend(uint8_t cmd, char *packet, uint8_t len) {
 *  @param
 *  @return
 ****************************************************************************************/
-static float convIntTofloat(char *data) {
-    union {
-      float f;
-      unsigned char byte[4];
-    } conv;
-
-    conv.byte[0] = data[0];
-    conv.byte[1] = data[1];
-    conv.byte[2] = data[2];
-    conv.byte[3] = data[3];
-
-    return conv.f;
+static float convIntTofloat(char *data, uint8_t len) {
+
+    float f;
+
+    memcpy(&f, data, len);
```

Comment:
Like you said, this isn't endianness-agnostic.

One could memcpy float -> uint32 and then do the uint32 endian-aware serialization?